docs/gds-supported-platform/tls-certificates.md

```diff
@@ -0,0 +1,36 @@
+# Public TLS Certificates
+
+TLS certificates can optionally be provisioned via [cert-manager][] in GSP.
+
+By default, the GSP has a `ClusterIssuer` named `letsencrypt-r53` that is configured to provision TLS certificates supplied by [LetsEncrypt][] via the DNS01 ACME challenge. For example: to provision a TLS certificate for the [gsp-canary][] in the sandbox cluster that is mounted into its local istio-ingressgateway:
+
+```yaml
```

Comment:
What exactly do you do with this code?

Reply:
Ideally a user would do nothing with this code. I've added it as an example of what a template's output might look like when it's rendered. I don't really want to include helm templates in these docs as I think that won't be as useful.

Should I perhaps rephrase the preceding sentence slightly:

> For example: a rendered template to provision a TLS certificate....

Reply:
well the user presumably needs to do something in order to achieve the stated task of

> to provision a TLS certificate for the [gsp-canary][] in the sandbox cluster that is mounted into its local istio-ingressgateway

what is it that they need to do?

Reply:
I've changed the phrasing and layout slightly to address this. Hopefully it's improved...?

Reply:
From a content point of view, the only other thing I would say is replace "For example:" with "For example," just to ensure we use colons in line with style.

If you are both happy that it is factually accurate, then LGTM.

Reply:
Updated.